torch/quantization/fuse_modules.py

```diff
@@ -45,12 +48,15 @@ def fuse_conv_bn_relu(conv, bn, relu):
     """
     assert(conv.training == bn.training == relu.training),\
         "Conv and BN both must be in the same mode (train or eval)."
-
+    is_3d = isinstance(conv, torch.nn.Conv3d)
```

Comment:
actually, do you need to verify Bn and Conv have matching dimensions here?

Reply:
I think we need to apply the same constraint as ConvBN.

Reply:
you mean ConvBn2d? yeah please add that as well